modules/update-upgrading-web.adoc

```diff
@@ -19,11 +20,12 @@ of the Customer Portal.
 
 . From the web console, click *Administration* > *Cluster Settings* and review
 the contents of the *Overview* tab.
-.. For production clusters, ensure that the *CHANNEL* is set to `stable-4.1`.
+.. For production clusters, ensure that the *CHANNEL* is set to the correct
+`stable-<major_version>` channel for the version that you want to update to.
```

Comment:
Is `{major.minor_version}` a thing?  Would be nice if you didn't have to bump this manually, but could get it automatically expanded to match whatever flavor the docs were being built for.

Reply:
It's not. I think that it's a good idea though, so I'm going to bring it through our internal change process.